blockdevice/blockdevice.go

```diff
@@ -159,6 +181,81 @@ type FileSystemInformation struct {
 	MountPoint []string
 }
 
+// CapacityInformation holds the capacity related information for the device
+type CapacityInformation struct {
```

Comment:
lets rename CapacityInformation to DeviceAttributes, then it makes sense to keep all of them in same struct. DeviceInformation will be used for identifier related information.

Reply:
We already have a `DeviceInformation` right. Shall we rename that to `DeviceAttributes` and move the blocksize and sector size to that struct. 

I prefer `CapacityInformation` because we need something that says about the actual capacity of the blockdevice. We will keep only `Storage` in `CapacityInformation`

For identifier related information we have an `Identifier` field.

Reply:
seems to be fine.

Reply:
```
type BlockDevice struct {
	Identifier
	.
	.
	.
	Capacity CapacityInformation
	DevLinks []DevLink
	DeviceDetails DeviceAttributes
}
type Identifier struct {
	UUID string
	SysPath string
	DevPath string
}
type CapacityInformation struct {
	Storage uint64
}
type DeviceAttributes struct {
	PhysicalBlockSize uint32
	LogicalBlockSize uint32
	HardwareSectorSize uint32
	DeviceType string
	DriveType string
	WWN string
	Vendor string
	Model string
	.
	.
}
```
This is how it will look after the change.

Reply:
Done the changes as discussed.

Reply:
@akhilerm do we need to update API struct also?